backend/logic/shop/config.js

```diff
@@ -40,6 +40,7 @@ const dbConfigOptionalKeys = [
   'deliveryApi',
   'email', // Email provider: 'aws', 'sendgrid', 'mailgun'
   'emailSubject',
+  'inventory',
```

Comment:
Add a comment explaining this is a boolean indicating if product inventory is enabled/disabled for the shop